lib/logstash/plugin_mixins/elasticsearch/common.rb

```diff
@@ -268,7 +268,7 @@ def submit(actions)
           @document_level_metrics.increment(:non_retryable_failures)
           @logger.warn "Failed action", status: status, action: action, response: response if log_failure_type?(error)
           next
-        elsif DOC_DLQ_CODES.include?(status)
+        elsif @dlq_codes.include?(status)
```

Comment:
It is causing red CI in integration test version 6.x

Reply:
the error is:
```
undefined method `include?' for nil:NilClass\nDid you mean?  include_class", :exception=>NoMethodError
```
So `@dlq_codes` is `nil` fo r `6.x`. I think it's related to the way `dlq_enabled?` works on LS `6.x`

Reply:
My guess was that the way `dlq_enabled?` operates on classes to determine if DLQ is active involved some fields that wasn't present on `6.8` but [ExecutionContextExt.dlqWriter](https://github.com/elastic/logstash/blob/50b43ebbaf35a042b9d1848118dcffde264dfbc9/logstash-core/src/main/java/org/logstash/execution/ExecutionContextExt.java#L39) exists in `6.8`